scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/exceptions/SpecificationLoadException.java

```diff
@@ -1,10 +1,9 @@
 package gov.nasa.jpl.aerie.scheduler.server.exceptions;
 
+import java.util.List;
```

Comment:
The "implement schedule workers" commit seems to have a broad swath of non-functional changes in it (imports reordering, marking classes as final, adding/removing whitespace). Would it be possible to split them out, to make the functional changes easier to review?

(Interestingly enough, your imports autosorted disagrees with my imports autosorter... sounds like we need to standardize 😅)

I took note of some of the files that I found with no functional changes, just import re-ordering. I think the commit would be a lot less intimidating-looking without them:
- SpecificationLoadException.java
- GraphQLParsers.java
- ActivityTemplateJSONParser.java
- InvalidEntityException.java
- SchedulerParsers.java
- SerializedValueJsonParser.java
- StrictSerializedValueJsonParser.java
- MerlinActivityInstance.java
- MerlinPlan.java
- PlanMetadata.java
- SchedulingCompilationError.java
- SchedulingDSL.java
- Specification.java
- CancelSchedulingRequestAction.java
- CreateRequestAction.java
- DeleteRequestAction.java
- GetCreatedActivitiesAction.java
- GetGoalSatisfaction.java
- GetRequestAction.java
- GetSatisfyingActivitiesAction.java
- GetSpecificationAction.java
- GetSpecificationConditionsAction.java
- GetSpecificationGoalsAction.java
- GoalBuilder.java
- InsertCreatedActivitiesAction.java
- InsertGoalSatisfactionAction.java
- InsertSatisfyingActivitiesAction.java
- PostgresParsers.java
- PostgresSpecificationRepository.java
- PreparedStatements.java
- SetRequestStateAction.java
- SpecificationRecord.java
- GraphQLMerlinService.java
- MissionModelService.java
- PlanService.java
- ThreadedSchedulerAgent.java
- GraphQLParsersTest.java

Reply:
Wow, thanks for taking the time to compile this! Just pushed a separate commit for this that precedes the `Implement scheduler workers` commit.

Reply:
Thanks for splitting it out - you've unblocked my ability to review this PR, so I'll try to get on that now :+1: 

I do wonder whether it would make sense to remove those changes from this PR entirely. So far, the only policy I'm aware of is that we like to avoid `.*` imports (which it looks like your import auto formatter inserted). We have talked about maybe making a style guide - if you feel strongly about import order, maybe you can help start drafting a style guide so we can start to standardize.

Reply:
Ahh I see. My intent here was to remove any unused imports, any stylistic changes were a side-effect, no strong opinion on import style here :)

Reply:
But looks like there are only a few files that had unused imports lol (like [GetRequestAction.java](https://github.com/NASA-AMMOS/aerie/pull/406/commits/c240ddccfb88c9f458516af173a380d0a146f504#diff-b19bd10556a8ec67369d66a5a8443a1f37ac6b7a8d3aa58d458f2b2e268a5d2c)). I'm happy to drop this commit altogether since it's 90% stylistic side-effects and 10% removing unused imports.

Reply:
Commit dropped now :)